git_scraper.py

```diff
@@ -0,0 +1,73 @@
+from bs4 import BeautifulSoup
+import urllib
+import requests
+import csv
+import constant
+import time
+
+
+def get_user_details(user):
+    req = requests.get(constant.USER_URL + user, headers={'User-Agent': "Magic Browser"})
+    soup = BeautifulSoup(req.content, "html.parser")
+
+    user_details = {}
+    repositories = soup.find('nav', {'class', 'UnderlineNav-body width-full p-responsive'}).span.text
+    try:
+        followers = soup.find('div', {'class', 'flex-order-1 flex-md-order-none mt-2 mt-md-0'}).span.text
+        user_details['FOLLOWERS'] = followers
+        user_details['REPOSITORIES'] = repositories
+    except AttributeError:
+        print(constant.ERROR)
+    return user_details
+
+
+def get_details(user):
+    user_name = user.find('a', {'class', 'color-fg-muted'})
+    try:
+        user_name = user_name.em.string
+    except AttributeError:
+        return
+    name = user.find('div', {'class', 'f4 text-normal'}).a.text.strip()
+    bio = user.find('p', class_='mb-1')
+
+    user_details = {'NAME': name, 'USER_NAME': user_name}
+    if bio:
+        user_details['BIO'] = bio.text.strip()
+    else:
+        user_details['BIO'] = constant.BIO
```

Comment:
For cases when values are not present ALWAYS provide null or None values rather than empty strings or other things such as "No Bio" or "Na"

This is a general programming approach. Makes it much easier to apply checks rather than match strings. String matching should be avoided. You may not be doing any string matching on it today but in future you might need to do something basis profiles who have bio vs those who dont